build.sbt

```diff
@@ -31,6 +31,23 @@ lazy val apso = (project in file("."))
   .dependsOn(akkaHttp, aws, caching, collections, core, elasticsearch, encryption, hashing, io, json, profiling, time)
   .aggregate(akkaHttp, aws, caching, collections, core, elasticsearch, encryption, hashing, io, json, profiling, testkit, time)
 
+
+lazy val docs = (project in file("apso-docs"))
+  .dependsOn(apso)
+  .settings(
+    mdocOut := baseDirectory.in(ThisBuild).value,
+
+    mdocVariables := Map(
+      "VERSION" -> "0.16.1" // This version should be set to the currently released version.
+    ),
+
+    libraryDependencies ++= Seq(Dependencies.AwsJavaSdkS3),
```

Comment:
Can you add a comment here stating that this is necessary because `aws-java-sdk-s3` has the `provided` scope in apso-aws?

Reply:
Done on b01c0f09cf6ba569e3284019e9e1f4ba3cde6070.